SQL/updates/30-31.py

```diff
@@ -0,0 +1,173 @@
+# Listen up.
+# In order to run this script on Windows, you need to make sure you have Python **3** installed. Tested on 3.8.2
+# It won't work on 2.7 at all.
+
+# To run this, supply the following args in a command shell
+# python 30-31.py address username password database table_old table_new
+# Example:
+# python 30-31.py localhost paradise ban_old ban
+
+# !/usr/bin/env python3
+import mysql.connector
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("address", help="MySQL server address (use localhost for the current computer)")
+parser.add_argument("username", help="MySQL login username")
+parser.add_argument("password", help="MySQL login password")
+parser.add_argument("database", help="Database name")
+parser.add_argument("curtable", help="Name of the current old ban table")
+parser.add_argument("newtable", help="Name of the new bab table to insert to, can't be same as the source table")
+parser.add_argument("port2server", help="Port to server name conversion instructions, expl: 7000:main,7001:secondary")
+parser.add_argument("fallbackserver", help="Server name that will be set if port not present in port2server list")
+
+args = parser.parse_args()
+db = mysql.connector.connect(host=args.address, user=args.username, passwd=args.password, db=args.database)
+cursor = db.cursor()
+current_table = args.curtable
+new_table = args.newtable
+port2server = {}
+fallback_server_name = args.fallbackserver
+
+for pair in args.port2server.split(","):
+    split = pair.split(":")
+    if len(split) > 1:
+        port2server[split[1]] = split[0]
```

Comment:
```suggestion
        port2server[split[0]] = split[1]
```

При аргументе `7721:main,7726:main,7729:main,7728:main,7724:ex,7723:wl,7725:sierra,7722:prime,7727:tgmc,7730:tgstation` получается такое

```Port main mapped to server 7728
Port ex mapped to server 7724
Port wl mapped to server 7723
Port sierra mapped to server 7725
Port prime mapped to server 7722
Port tgmc mapped to server 7727
Port tgstation mapped to server 7730
Fallback server other```